src/common/meta/src/cluster.rs

```diff
@@ -100,6 +106,19 @@ pub enum NodeStatus {
     Datanode(DatanodeStatus),
     Frontend(FrontendStatus),
     Metasrv(MetasrvStatus),
+    Standalone,
```

Comment:
consider include these status as a json field in cluster_info.

Reply:
It will be in another table such as `cluster_stats` etc. The `cluster_info` only provides the topology info.